create a epub standalone

pedrorobsonleao opened the pull request:
create a standalone epub using **pandoc** comand in Linux plataform.

pedro-leao-apontador committed:
create a epub

snyk-bot committed:
fix: package.json & .snyk to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:q-io:20180212
- https://snyk.io/vuln/npm:qs:20170213


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/npm:hoek:20180212
- https://snyk.io/vuln/npm:lodash:20180130
- https://snyk.io/vuln/npm:request:20160119
- https://snyk.io/vuln/npm:tough-cookie:20170905
- https://snyk.io/vuln/npm:tunnel-agent:20170305

pedrorobsonleao committed:
Merge pull request #1 from pedrorobsonleao/snyk-fix-369edaf8f14d84a428d7bda3cb16b674

[Snyk] Fix for 7 vulnerabilities

snyk-bot committed:
fix: package.json & .snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-HTTPSPROXYAGENT-469131

pedrorobsonleao committed:
Merge pull request #2 from pedrorobsonleao/snyk-fix-bffad8d638c1c8bf666077bccc120fba

[Snyk] Fix for 1 vulnerabilities

Matheus-de-Souza committed:
Corrigir link para a imagem busybox

somatorio committed:
Merge pull request #107 from Matheus-de-Souza/patch-1

Corrigir link para a imagem busybox

Commit:
Correção do nome e da logo da DigitalOcean

somatorio committed:
Merge pull request #108 from fcpimenta/digitalocean

Correção do nome e da logo da DigitalOcean

snyk-bot committed:
fix: package.json & .snyk to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:q-io:20180212
- https://snyk.io/vuln/npm:qs:20170213


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/npm:hoek:20180212
- https://snyk.io/vuln/npm:lodash:20180130
- https://snyk.io/vuln/npm:request:20160119
- https://snyk.io/vuln/npm:tough-cookie:20170905
- https://snyk.io/vuln/npm:tunnel-agent:20170305

pedrorobsonleao committed:
Merge pull request #5 from pedrorobsonleao/snyk-fix-9ada946eeb6993f58ac59bcbeea16f41

[Snyk] Security upgrade gh-pages from 0.12.0 to 1.0.0

somatorio commented:
Não está com cara de que iria quebrar o sistema da leanpub, mas só pra garantir... chegou a verificar se conflita em algo com o sistema deles?

robsonpeixoto committed:
Adicionar o link do leanpub

somatorio committed:
Merge pull request #109 from robsonpeixoto/patch-1

Adicionar o link do leanpub

pedrorobsonleao committed:
fix: change pandoc parameter

pedrorobsonleao committed:
fix: rebase

pedrorobsonleao committed:
Merge branch 'gomex:master' into master

snyk-bot committed:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-QS-3153490

Commit:
Correção de comando que não estava formato e ao gerar o epub ocasionava erro no arquivo gerado

somatorio committed:
Merge pull request #111 from rlresende/master

Correção no capítulo 14

pedrorobsonleao committed:
Merge pull request #8 from pedrorobsonleao/snyk-fix-d3ac3e73d57c7e9386285b873e73abee

[Snyk] Security upgrade gh-pages from 0.12.0 to 1.0.0

somatorio commented:
@pedro-leao-apontador Chegou a verificar o que pedi? :x

somatorio commented:
@pedrorobsonleao chegou a verificar o que eu pedi? :x

IvanCCO committed:
Correção de palavra incompleta

somatorio committed:
Merge pull request #112 from IvanCCO/patch-1

Correção de palavra incompleta

pedrorobsonleao committed:
Merge branch 'gomex:master' into master

pedrorobsonleao committed:
Create makefile.yml

pedrorobsonleao committed:
Update makefile.yml

remove configure

pedrorobsonleao committed:
Update makefile.yml

install pandoc

pedrorobsonleao committed:
Update makefile.yml

pedrorobsonleao committed:
Update Makefile

pedrorobsonleao committed:
Update Makefile

pedrorobsonleao committed:
Update makefile.yml

pedrorobsonleao committed:
Update Makefile

somatorio committed:
Merge branch 'main' into master
